<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIABuildDialog</class>
 <widget class="QDialog" name="CIABuildDialog">
  <property name="windowTitle">
   <string>Build CIA</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="destination"/>
     </item>
     <item>
      <widget class="QToolButton" name="destinationExplore">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox">
     <property name="title">
      <string>Build Type</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QRadioButton" name="standardButton">
        <property name="text">
         <string>Standard</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel">
        <property name="text">
         <string>Recommended for general use.&lt;br&gt;Decrypted CIA with decrypted contents and standard ticket.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pirateLegitButton">
        <property name="text">
         <string>Legit with standard ticket ("pirate legit")</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pirateLegitLabel">
        <property name="text">
         <string>Encrypted CIA with legit TMD, encrypted contents and standard ticket.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="legitButton">
        <property name="text">
         <string>Legit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="legitLabel">
        <property name="text">
         <string>Encrypted CIA with legit TMD, encrypted contents and legit ticket.&lt;br&gt;WARNING: Legit ticket may include console identifying information!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
